test/models/fund_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class FundTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
I do want to add some actual scoping tests here to prevent regression; one of those 'fix-it-in-post' kinda things. I'll get to it when we've agreed on the general broad outline here.